### ISSUE TITLE
an small convertion issue in `callout` blocks

### ISSUE THREAD

**Issue by adehghanzadeh** [souvikinator/notion-to-md#48]
Hi :)
I found another small issue
Let me show you some screenshots

Notion:
![Screenshot_20220802_232634](https://user-images.githubusercontent.com/86428901/182453615-598244a6-6901-4ee7-bc30-0d23aaa1ba8d.png)

Result:
![Screenshot_20220802_231555](https://user-images.githubusercontent.com/86428901/182453656-4faa29b2-d896-4957-bdf8-47e3264ee324.png)
code:
![Screenshot_20220802_230846](https://user-images.githubusercontent.com/86428901/182453682-25e7f945-f300-4f04-9dbf-5b027a756f7f.png)
right here, the four-space before image without `>`

----
Expectation:
![Screenshot_20220802_231604](https://user-images.githubusercontent.com/86428901/182453736-ddcc3679-1d3a-44ab-ab78-7d1dbe02f36b.png)
manually edited code:
![Screenshot_20220802_233409](https://user-images.githubusercontent.com/86428901/182453790-1f18de52-77a8-460e-be99-639cd2734bf1.png)

same in github (markdown), its not a Jekyll-specific problem
![Screenshot_20220802_234412](https://user-images.githubusercontent.com/86428901/182455409-27e1e2d6-957d-4df8-8c9f-6dbdc0fb4d54.png)

thanks in advance

**Comment by adehghanzadeh** [comment]
Update
Same issue on quotes

I guess its because of an empty line between first and other line of quotation

## without blank line:
> this is first line
its second line



```text
> this is first line
its second line
```
---
## with a blank line:
> its first line

its second line



```text
> its first line

its second line
```

---

note that, in my screenshots above, I used a `>` to put `Github Secret` image on quote but I realized that its better to remove `>` and the space afterward (to make sure that Jekyll wont create a code-block)

**Comment by souvikinator** [comment]
I'll fix them asap and it should be live in the next release.

**Comment by souvikinator** [comment]
The callout issue is not fixed and is live in `v2.5.5`.
Closing the issue now. Feel free to reopen the issue if any problem persists.
Thank you for the contribution.

**Comment by adehghanzadeh** [comment]
So, for a block line in quote, a `>` should be write at first of each line

second, I used quote combined with list and the output markdown is:

```markdown
> Updates:

	- use `publishTime` instead on `created_time` so the time can be modified by a property
	- use `topCategories` and `subCategories` to categorize posts (note that my theme support only one-step subcategory)
	- use `mathCheck` and `mermaidCheck` to enable Math and Mermaid support in posts
```


### preview:
> Updates:

	- use `publishTime` instead on `created_time` so the time can be modified by a property
	- use `topCategories` and `subCategories` to categorize posts (note that my theme support only one-step subcategory)
	- use `mathCheck` and `mermaidCheck` to enable Math and Mermaid support in posts

---

## Edited
I edited this to (remove blank line)
```markdown
> Updates:
	- use `publishTime` instead on `created_time` so the time can be modified by a property
	- use `topCategories` and `subCategories` to categorize posts (note that my theme support only one-step subcategory)
	- use `mathCheck` and `mermaidCheck` to enable Math and Mermaid support in posts
```

### preview:

> Updates:
	- use `publishTime` instead on `created_time` so the time can be modified by a property
	- use `topCategories` and `subCategories` to categorize posts (note that my theme support only one-step subcategory)
	- use `mathCheck` and `mermaidCheck` to enable Math and Mermaid support in posts
